src/account/operations/transfer/high_level/burn_nft.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use iota_client::bee_message::{
+    address::{Address, NftAddress},
+    output::{BasicOutputBuilder, NftId, Output, OutputId},
+};
+
+use crate::{
+    account::{handle::AccountHandle, operations::transfer::TransferResult, TransferOptions},
+    Error,
+};
+
+impl AccountHandle {
+    /// Function to mint nft.
```

Comment:
Should be burn or destroy, here it says https://github.com/lzpap/tips/blob/master/tips/TIP-0018/tip-0018.md#consumed-outputs-2 burning 🤔

Reply:
No, should stay